frontend/unity/Build/unity.json

```diff
@@ -5,7 +5,7 @@
 "dataUrl": "unity.data.unityweb",
 "wasmCodeUrl": "unity.wasm.code.unityweb",
 "wasmFrameworkUrl": "unity.wasm.framework.unityweb",
-"graphicsAPI": ["WebGL 2.0"],
+"graphicsAPI": ["WebGL 2.0","WebGL 1.0"],
```

Comment:
@carmenfan From what I can see we now support viewing of models for browser with WebGL 1.0 as well.
I checked it on Safari and it works 🎉 
Should we, therefore, update the support check for WebGL 2.0, which I implemented some time ago?
```
Your browser does not support WebGL 2.0 therefore the 3D Viewer will be unavailable. However, you can still other functionalities we offer.
```

Reply:
@m1chalkubiak  good point!